2015/11/15 12:13:42 [error] [exception.CException] exception 'CException' with message '属性 "User.name" 未被定义.' in /home/admin/works/urtime/framework/base/CComponent.php:130
Stack trace:
#0 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(145): CComponent->__get('name')
#1 /home/admin/works/urtime/protected/services/v0/AllianceService.php(78): CActiveRecord->__get('name')
#2 /home/admin/works/urtime/protected/controllers/IndexController.php(92): AllianceService->getAlliances(Array)
#3 /home/admin/works/urtime/framework/web/actions/CInlineAction.php(49): IndexController->actionIndex()
#4 /home/admin/works/urtime/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#5 /home/admin/works/urtime/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#6 /home/admin/works/urtime/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 /home/admin/works/urtime/framework/web/CWebApplication.php(282): CController->run('index')
#8 /home/admin/works/urtime/framework/web/CWebApplication.php(141): CWebApplication->runController('index/index')
#9 /home/admin/works/urtime/framework/base/CApplication.php(184): CWebApplication->processRequest()
#10 /home/admin/works/urtime/index.php(35): CApplication->run()
#11 {main}
REQUEST_URI=/index/index.html?debug=1
HTTP_REFERER=http://local.urtime.com/req?action=getAlliances
---
2015/11/15 12:15:27 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in where clause is ambiguous. The SQL statement executed was: SELECT `t`.`id` AS `t0_c0`, `t`.`name` AS `t0_c1`, `t`.`leader` AS `t0_c2`, `t`.`image` AS `t0_c3`, `t`.`type` AS `t0_c4`, `t`.`center_name` AS `t0_c5`, `t`.`accept` AS `t0_c6`, `t`.`notice` AS `t0_c7`, `t`.`gmt_created` AS `t0_c8`, `t`.`gmt_modified` AS `t0_c9`, `user`.`id` AS `t1_c0`, `user`.`nickname` AS `t1_c1`, `user`.`username` AS `t1_c2`, `user`.`mobile` AS `t1_c3`, `user`.`email` AS `t1_c4`, `user`.`sex` AS `t1_c5`, `user`.`image` AS `t1_c6`, `user`.`province` AS `t1_c7`, `user`.`city` AS `t1_c8`, `user`.`gmt_created` AS `t1_c9`, `user`.`gmt_modified` AS `t1_c10`, `user`.`password` AS `t1_c11` FROM `t_alliance` `t`  LEFT OUTER JOIN `t_user` `user` ON (`t`.`leader`=`user`.`id`) WHERE (id=:ycp0).
in /home/admin/works/urtime/protected/models/Alliance.php (135)
2015/11/15 12:15:27 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in where clause is ambiguous. The SQL statement executed was: SELECT `t`.`id` AS `t0_c0`, `t`.`name` AS `t0_c1`, `t`.`leader` AS `t0_c2`, `t`.`image` AS `t0_c3`, `t`.`type` AS `t0_c4`, `t`.`center_name` AS `t0_c5`, `t`.`accept` AS `t0_c6`, `t`.`notice` AS `t0_c7`, `t`.`gmt_created` AS `t0_c8`, `t`.`gmt_modified` AS `t0_c9`, `user`.`id` AS `t1_c0`, `user`.`nickname` AS `t1_c1`, `user`.`username` AS `t1_c2`, `user`.`mobile` AS `t1_c3`, `user`.`email` AS `t1_c4`, `user`.`sex` AS `t1_c5`, `user`.`image` AS `t1_c6`, `user`.`province` AS `t1_c7`, `user`.`city` AS `t1_c8`, `user`.`gmt_created` AS `t1_c9`, `user`.`gmt_modified` AS `t1_c10`, `user`.`password` AS `t1_c11` FROM `t_alliance` `t`  LEFT OUTER JOIN `t_user` `user` ON (`t`.`leader`=`user`.`id`) WHERE (id=:ycp0)' in /home/admin/works/urtime/framework/db/CDbCommand.php:543
Stack trace:
#0 /home/admin/works/urtime/framework/db/CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(808): CDbCommand->queryAll()
#2 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(440): CJoinElement->runQuery(Object(CJoinQuery))
#3 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(68): CJoinElement->find(Object(CDbCriteria))
#4 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1361): CActiveFinder->query(Object(CDbCriteria), false)
#5 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1461): CActiveRecord->query(Object(CDbCriteria))
#6 /home/admin/works/urtime/protected/models/Alliance.php(135): CActiveRecord->find(Object(CDbCriteria))
#7 /home/admin/works/urtime/protected/services/v0/AllianceService.php(59): Alliance->getAlliance('1')
#8 /home/admin/works/urtime/protected/controllers/IndexController.php(92): AllianceService->getAlliances(Array)
#9 /home/admin/works/urtime/framework/web/actions/CInlineAction.php(49): IndexController->actionIndex()
#10 /home/admin/works/urtime/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /home/admin/works/urtime/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /home/admin/works/urtime/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /home/admin/works/urtime/framework/web/CWebApplication.php(282): CController->run('index')
#14 /home/admin/works/urtime/framework/web/CWebApplication.php(141): CWebApplication->runController('index/index')
#15 /home/admin/works/urtime/framework/base/CApplication.php(184): CWebApplication->processRequest()
#16 /home/admin/works/urtime/index.php(35): CApplication->run()
#17 {main}
REQUEST_URI=/index/index.html?debug=1
HTTP_REFERER=http://local.urtime.com/req?action=getAlliances
---
2015/11/15 12:17:40 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in where clause is ambiguous. The SQL statement executed was: SELECT `t`.`id` AS `t0_c0`, `t`.`name` AS `t0_c1`, `t`.`leader` AS `t0_c2`, `t`.`image` AS `t0_c3`, `t`.`type` AS `t0_c4`, `t`.`center_name` AS `t0_c5`, `t`.`accept` AS `t0_c6`, `t`.`notice` AS `t0_c7`, `t`.`gmt_created` AS `t0_c8`, `t`.`gmt_modified` AS `t0_c9`, `user`.`id` AS `t1_c0`, `user`.`nickname` AS `t1_c1`, `user`.`username` AS `t1_c2`, `user`.`mobile` AS `t1_c3`, `user`.`email` AS `t1_c4`, `user`.`sex` AS `t1_c5`, `user`.`image` AS `t1_c6`, `user`.`province` AS `t1_c7`, `user`.`city` AS `t1_c8`, `user`.`gmt_created` AS `t1_c9`, `user`.`gmt_modified` AS `t1_c10`, `user`.`password` AS `t1_c11` FROM `t_alliance` `t`  LEFT OUTER JOIN `t_user` `user` ON (`t`.`leader`=`user`.`id`) WHERE (id=:ycp0).
in /home/admin/works/urtime/protected/models/Alliance.php (135)
2015/11/15 12:17:40 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in where clause is ambiguous. The SQL statement executed was: SELECT `t`.`id` AS `t0_c0`, `t`.`name` AS `t0_c1`, `t`.`leader` AS `t0_c2`, `t`.`image` AS `t0_c3`, `t`.`type` AS `t0_c4`, `t`.`center_name` AS `t0_c5`, `t`.`accept` AS `t0_c6`, `t`.`notice` AS `t0_c7`, `t`.`gmt_created` AS `t0_c8`, `t`.`gmt_modified` AS `t0_c9`, `user`.`id` AS `t1_c0`, `user`.`nickname` AS `t1_c1`, `user`.`username` AS `t1_c2`, `user`.`mobile` AS `t1_c3`, `user`.`email` AS `t1_c4`, `user`.`sex` AS `t1_c5`, `user`.`image` AS `t1_c6`, `user`.`province` AS `t1_c7`, `user`.`city` AS `t1_c8`, `user`.`gmt_created` AS `t1_c9`, `user`.`gmt_modified` AS `t1_c10`, `user`.`password` AS `t1_c11` FROM `t_alliance` `t`  LEFT OUTER JOIN `t_user` `user` ON (`t`.`leader`=`user`.`id`) WHERE (id=:ycp0)' in /home/admin/works/urtime/framework/db/CDbCommand.php:543
Stack trace:
#0 /home/admin/works/urtime/framework/db/CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(808): CDbCommand->queryAll()
#2 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(440): CJoinElement->runQuery(Object(CJoinQuery))
#3 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(68): CJoinElement->find(Object(CDbCriteria))
#4 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1361): CActiveFinder->query(Object(CDbCriteria), false)
#5 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1461): CActiveRecord->query(Object(CDbCriteria))
#6 /home/admin/works/urtime/protected/models/Alliance.php(135): CActiveRecord->find(Object(CDbCriteria))
#7 /home/admin/works/urtime/protected/services/v0/AllianceService.php(60): Alliance->getAlliance('1')
#8 /home/admin/works/urtime/protected/controllers/IndexController.php(92): AllianceService->getAlliances(Array)
#9 /home/admin/works/urtime/framework/web/actions/CInlineAction.php(49): IndexController->actionIndex()
#10 /home/admin/works/urtime/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /home/admin/works/urtime/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /home/admin/works/urtime/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /home/admin/works/urtime/framework/web/CWebApplication.php(282): CController->run('index')
#14 /home/admin/works/urtime/framework/web/CWebApplication.php(141): CWebApplication->runController('index/index')
#15 /home/admin/works/urtime/framework/base/CApplication.php(184): CWebApplication->processRequest()
#16 /home/admin/works/urtime/index.php(35): CApplication->run()
#17 {main}
REQUEST_URI=/index/index.html?debug=1
HTTP_REFERER=http://local.urtime.com/req?action=getAlliances
---
2015/11/15 12:18:34 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in where clause is ambiguous. The SQL statement executed was: SELECT `t`.`id` AS `t0_c0`, `t`.`name` AS `t0_c1`, `t`.`leader` AS `t0_c2`, `t`.`image` AS `t0_c3`, `t`.`type` AS `t0_c4`, `t`.`center_name` AS `t0_c5`, `t`.`accept` AS `t0_c6`, `t`.`notice` AS `t0_c7`, `t`.`gmt_created` AS `t0_c8`, `t`.`gmt_modified` AS `t0_c9`, `user`.`id` AS `t1_c0`, `user`.`nickname` AS `t1_c1`, `user`.`username` AS `t1_c2`, `user`.`mobile` AS `t1_c3`, `user`.`email` AS `t1_c4`, `user`.`sex` AS `t1_c5`, `user`.`image` AS `t1_c6`, `user`.`province` AS `t1_c7`, `user`.`city` AS `t1_c8`, `user`.`gmt_created` AS `t1_c9`, `user`.`gmt_modified` AS `t1_c10`, `user`.`password` AS `t1_c11` FROM `t_alliance` `t`  LEFT OUTER JOIN `t_user` `user` ON (`t`.`leader`=`user`.`id`) WHERE (id=:ycp0).
in /home/admin/works/urtime/protected/models/Alliance.php (136)
2015/11/15 12:18:34 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'id' in where clause is ambiguous. The SQL statement executed was: SELECT `t`.`id` AS `t0_c0`, `t`.`name` AS `t0_c1`, `t`.`leader` AS `t0_c2`, `t`.`image` AS `t0_c3`, `t`.`type` AS `t0_c4`, `t`.`center_name` AS `t0_c5`, `t`.`accept` AS `t0_c6`, `t`.`notice` AS `t0_c7`, `t`.`gmt_created` AS `t0_c8`, `t`.`gmt_modified` AS `t0_c9`, `user`.`id` AS `t1_c0`, `user`.`nickname` AS `t1_c1`, `user`.`username` AS `t1_c2`, `user`.`mobile` AS `t1_c3`, `user`.`email` AS `t1_c4`, `user`.`sex` AS `t1_c5`, `user`.`image` AS `t1_c6`, `user`.`province` AS `t1_c7`, `user`.`city` AS `t1_c8`, `user`.`gmt_created` AS `t1_c9`, `user`.`gmt_modified` AS `t1_c10`, `user`.`password` AS `t1_c11` FROM `t_alliance` `t`  LEFT OUTER JOIN `t_user` `user` ON (`t`.`leader`=`user`.`id`) WHERE (id=:ycp0)' in /home/admin/works/urtime/framework/db/CDbCommand.php:543
Stack trace:
#0 /home/admin/works/urtime/framework/db/CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(808): CDbCommand->queryAll()
#2 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(440): CJoinElement->runQuery(Object(CJoinQuery))
#3 /home/admin/works/urtime/framework/db/ar/CActiveFinder.php(68): CJoinElement->find(Object(CDbCriteria))
#4 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1361): CActiveFinder->query(Object(CDbCriteria), false)
#5 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1461): CActiveRecord->query(Object(CDbCriteria))
#6 /home/admin/works/urtime/protected/models/Alliance.php(136): CActiveRecord->find(Object(CDbCriteria))
#7 /home/admin/works/urtime/protected/services/v0/AllianceService.php(60): Alliance->getAlliance('1')
#8 /home/admin/works/urtime/protected/controllers/IndexController.php(92): AllianceService->getAlliances(Array)
#9 /home/admin/works/urtime/framework/web/actions/CInlineAction.php(49): IndexController->actionIndex()
#10 /home/admin/works/urtime/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /home/admin/works/urtime/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#12 /home/admin/works/urtime/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#13 /home/admin/works/urtime/framework/web/CWebApplication.php(282): CController->run('index')
#14 /home/admin/works/urtime/framework/web/CWebApplication.php(141): CWebApplication->runController('index/index')
#15 /home/admin/works/urtime/framework/base/CApplication.php(184): CWebApplication->processRequest()
#16 /home/admin/works/urtime/index.php(35): CApplication->run()
#17 {main}
REQUEST_URI=/index/index.html?debug=1
HTTP_REFERER=http://local.urtime.com/req?action=getAlliances
---
2015/11/15 12:50:21 [error] [system.db.CDbCommand] CDbCommand::fetchAll() failed: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'alliance.apply_type' in 'where clause'. The SQL statement executed was: SELECT * FROM `t_relations` `t` WHERE (alliance_id=:ycp1) AND (alliance.apply_type=1) ORDER BY t.gmt_created asc.
in /home/admin/works/urtime/protected/models/Relations.php (120)
2015/11/15 12:50:21 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand 无法执行 SQL 语句: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'alliance.apply_type' in 'where clause'. The SQL statement executed was: SELECT * FROM `t_relations` `t` WHERE (alliance_id=:ycp1) AND (alliance.apply_type=1) ORDER BY t.gmt_created asc' in /home/admin/works/urtime/framework/db/CDbCommand.php:543
Stack trace:
#0 /home/admin/works/urtime/framework/db/CDbCommand.php(396): CDbCommand->queryInternal('fetchAll', Array, Array)
#1 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1356): CDbCommand->queryAll()
#2 /home/admin/works/urtime/framework/db/ar/CActiveRecord.php(1475): CActiveRecord->query(Object(CDbCriteria), true)
#3 /home/admin/works/urtime/protected/models/Relations.php(120): CActiveRecord->findAll(Object(CDbCriteria))
#4 /home/admin/works/urtime/protected/models/Alliance.php(146): Relations->getMembers('1', 0)
#5 /home/admin/works/urtime/protected/services/v0/AllianceService.php(60): Alliance->getAlliance('1')
#6 /home/admin/works/urtime/protected/controllers/IndexController.php(92): AllianceService->getAlliances(Array)
#7 /home/admin/works/urtime/framework/web/actions/CInlineAction.php(49): IndexController->actionIndex()
#8 /home/admin/works/urtime/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#9 /home/admin/works/urtime/framework/web/CController.php(286): CController->runAction(Object(CInlineAction))
#10 /home/admin/works/urtime/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 /home/admin/works/urtime/framework/web/CWebApplication.php(282): CController->run('index')
#12 /home/admin/works/urtime/framework/web/CWebApplication.php(141): CWebApplication->runController('index/index')
#13 /home/admin/works/urtime/framework/base/CApplication.php(184): CWebApplication->processRequest()
#14 /home/admin/works/urtime/index.php(35): CApplication->run()
#15 {main}
REQUEST_URI=/index/index.html?debug=1
HTTP_REFERER=http://local.urtime.com/req?action=getAlliances
---
